pkgs/development/libraries/vaapi-vdpau/default.nix

```diff
@@ -13,23 +8,25 @@ stdenv.mkDerivation rec {
     sha256 = "1fcvgshzyc50yb8qqm6v6wn23ghimay23ci0p8sm8gxcy211jp0m";
   };
 
-  patches = [ ./glext85.patch
-              (fetchurl { url = "http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/x11-libs/libva-vdpau-driver/files/libva-vdpau-driver-0.7.4-VAEncH264VUIBufferType.patch?revision=1.1";
-                          name = "libva-vdpau-driver-0.7.4-VAEncH264VUIBufferType.patch";
-                          sha256 = "166svcav6axkrlb3i4rbf6dkwjnqdf69xw339az1f5yabj72pqqs";
-                        }) ];
+  patches = [
+    ./libva-vdpau-driver-0.7.4-glext-85.patch
```

Comment:
Was it not possible to fetch these patches from anywhere?

Reply:
To be honest, I don't understand the preference to fetch the patches from elsewhere. FWIW, I pulled all of the patches from Fedora except the last one, from Gentoo.

Reply:
Of course, they were being fetched before, so I can update it if necessary....

Reply:
I think the problem it's a problem that comes with nixpkgs being a monorepo.
In fedora it's a separate repo for each package so it's fine, but for nixpkgs we'll make the repo size simply huge for everyone.

Reply:
Okay, changed to `fetchpatch` and pushed.